signature/src/policy/mod.rs

```diff
@@ -70,6 +70,18 @@ impl Policy {
         Ok(())
     }
 
+    // Get the set of signature schemes that need to be verified of the image.
+    pub fn get_signature_schemes(&self, image: &image::Image) -> Vec<Option<String>> {
```

Comment:
We should define an enum for each scheme that we want to support, instead of having it as a free form string.